src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt

```diff
@@ -11,22 +11,21 @@ import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.welcometoprison.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.Prison
 import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.PrisonService
 
 @RestController
 @Validated
-@PreAuthorize("hasRole('ROLE_VIEW_INCOMING_MOVEMENTS')")
-@RequestMapping(name = "Prison", path = ["/prison"], produces = [MediaType.APPLICATION_JSON_VALUE])
+@PreAuthorize("hasRole('ROLE_VIEW_ARRIVALS')")
```

Comment:
fixing role! this was broken and we weren't getting images on the front end because of it